rotkehlchen/inquirer.py

```diff
@@ -722,14 +718,14 @@ def _find_usd_price(
         )
         return price, oracle, used_main_currency
 
-    def find_uniswap_v2_lp_price(
+    def find_lp_price_from_pool_contract(
```

Comment:
As Yabir also mentioned before. Use different name.
```suggestion
    def find_lp_price_from_uniswaplike_pool(
```